galite-core/src/main/kotlin/org/kopi/galite/visual/ApplicationContext.kt

```diff
@@ -200,17 +200,17 @@ abstract class ApplicationContext {
           } catch (e: PropertyException) {
             null
           }
-          val sender: String = try {
-            getDefaults().getStringFor("debugging.mail.sender").toString()
+          val sender: String? = try {
+            getDefaults().getStringFor("debugging.mail.sender")
           } catch (e: PropertyException) {
             TODO()
           }
```

Comment:
I think that sender is not nullable but getStringFor("debugging.mail.sender") couldn't return nullable value